packages/bind-mount/package.json

```diff
@@ -14,7 +15,13 @@
     "node-gyp": "^11.0.0"
   },
   "scripts": {
-    "build": "node-gyp rebuild",
+    "build": "tsc && node-gyp rebuild",
     "install": "node-gyp rebuild"
+  },
+  "devDependencies": {
```

Comment:
Nit: can we use a more conventional order for `dependencies`/`scripts`/`devDependencies`? Conventionally, `scripts` comes first, then `dependencies`, then `devDependencies`.